ixdjango/docker_settings.py

```diff
@@ -23,7 +23,6 @@
 
 # Debugging
 DEBUG = ENVIRONMENT in ('dev_local', 'dev', 'test')
-TEMPLATE_DEBUG = DEBUG
```

Comment:
Does this package already set the post 1.8 TEMPLATE["OPTIONS"]["debug"] setting?

Reply:
No, the apps do that themselves.

Reply:
yeh thats fine, since no new apps will use this anyway.